lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -48,8 +54,45 @@ table.ccx-schedule {
 
 form.ccx-form {
     line-height: 1.5;
+    // inspiration was taken from https://github.com/edx/ux-pattern-library
     select {
         width: 100%;
+        height: ($baseline*1.9);
+        box-sizing: padding-box;
+        line-height: ($baseline*1.25);
+        transition: all 125ms ease-in-out 0s;
+        display: inline-block;
+        padding: ($baseline*.625);
+        border: 1px solid #e9e8e8;
+        background: #fcfcfc;
+        font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
I don't think you need to include this font-stack here. It should already be defined at a much higher level within the platform's CSS.

Please remove.